vscode_extension/src/providers/AacDefinitionsViewProvider.ts

```diff
@@ -0,0 +1,94 @@
+import * as vscode from 'vscode';
+import { aacRestApi, DefinitionModel } from "../requests/aacRequests";
+
+export class AacDefinitionsViewProvider implements vscode.TreeDataProvider<Definition> {
+
+    public COMMAND_DEFINITIONS_LIST_REFRESH = "definitionsList.refreshEntry";
+    public COMMAND_DEFINITIONS_LIST_EDIT = "definitionsList.editEntry";
+    public COMMAND_DEFINITIONS_LIST_DELETE = "definitionsList.deleteEntry";
+	private _onDidChangeTreeData: vscode.EventEmitter<Definition | undefined | void> = new vscode.EventEmitter<Definition | undefined | void>();
+	readonly onDidChangeTreeData: vscode.Event<Definition | undefined | void> = this._onDidChangeTreeData.event;
+
+    constructor() { }
+
+    getTreeItem(element: Definition): vscode.TreeItem {
+        return element;
+    }
+
+    getChildren(element?: Definition): Thenable<Definition[]> {
+        if (element) {
+            return Promise.resolve(
+                this.getDefinitionsInContext()
+            );
+        } else {
+            return Promise.resolve(this.getDefinitionsInContext());
+        }
+    }
+
+    refresh(): void {
+        this._onDidChangeTreeData.fire();
+    }
```

Comment:
While not required by TS because `public` is the default, it may make sense to prefix these methods with the `public` access modifier for consistency across our TS files.

Reply:
Done